src/dashboard/Settings/actions.js

```diff
@@ -0,0 +1,62 @@
+/* Actions for the incident */ /* const actions */
+export const ADD_INCIDENT_TYPE = 'INCIDENTS_TYPE:ADD_NEW_INCIDENT_TYPE';
+export const SELECT_INCIDENT_TYPE = 'INCIDENTS_TYPE :SELECTED_INCIDENTS_TYPE ';
+export const GET_INCIDENTS_TYPE = 'INCIDENTS_TYPE :TRIGER_GET';
+export const STORE_INCIDENTS_TYPE_SUCCESS =
+  'INCIDENTS_TYPE:STORE_INCIDENTS_TYPE_SUCCESS';
+export const UPDATE_INCIDENT_TYPE = 'INCIDENTS_TYPE :UPDATE_INCIDENT_TYPE';
+export const SELECT_COLOR_AUTOFILL = 'INCIDENTS_TYPE :COLOR_AUTOFILL';
+export const SEARCH_INCIDENT_TYPE = 'INCIDENTS_TYPE :SEARCH_INCIDENT_TYPE';
+export const FETCH_INCIDENT_TYPE_FAILURE =
+  'INCIDENTS_TYPE :FETCH_INCIDENT_TYPE_FAILURE';
+
+/* Actions creater */
```

Comment:
Also documentation in this file is not enough but it can wait(since there will be time for testing and documentation)